src/drawer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rmwc/drawer",
-  "version": "8.0.7",
+  "version": "8.0.8",
```

Comment:
Probably we don't want this on next, as we are doing a prerelease. Seems like my commit somehow made it into this PR